lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.CoolingTowerVariableSpeed.rb

```diff
@@ -0,0 +1,14 @@
+class ASHRAE901PRM < Standard
```

Comment:
This seems to be identical to the standards version of this file. Do we need to duplicate this?

Reply:
Removed the function in this file, but kept this file.